.eslintrc.yml

```diff
@@ -28,26 +27,38 @@ rules:
   eqeqeq: error
   handle-callback-err: error
   indent: [error, tab]
-  key-spacing: [error, {beforeColon: false, afterColon: true}]
-  keyword-spacing: [error, {before: true, after: true}]
+  key-spacing:
+    - error
+    - beforeColon: false
+      afterColon: true
```

Comment:
This looks a little odd not having the `-` before each line

Reply:
It's not stylistic, it's how (multiline) lists of dictionaries are done in YAML like [here](https://en.wikipedia.org/wiki/YAML#Example). Adding a `-` would be incorrect and ESLint would choke on it.
We already do this with `space-before-function-paren` for instance:

```yaml
  space-before-function-paren:
    - error
    - anonymous: never
      named: never
      asyncArrow: always # Otherwise requires `async()`
```

Reply:
Oh I see, it's not a list of 3 items, it's a list of 2 items, one item is then an object containing 2 items. That's much less obvious in the new syntax, whereas the old syntax made that quite clear. Can we not wrap the object with the curly braces like we do in the before?

Reply:
We can, but despite the `-`, I find:

```yaml
key-spacing:
    - error
    - beforeColon: false
      afterColon: true
```

less confusing than:

```yaml
key-spacing:
  - error
  - {beforeColon: false, afterColon: true}
```

Also, I notice that editors have a harder time coloring the second line, e.g. GitHub's color syntax (Atom as well, I don't know about VS Code).
Up to you though, I don't think this is really important so I'm happy to go either way 😅